up42/job.py

```diff
@@ -42,41 +43,51 @@ def __init__(
         self.results = None
         self.order_ids = order_ids
         if self.auth.get_info:
-            self.info = self._get_info()
+            self._info = self.info
```

Comment:
Do we need to define _info as an instance variable? Would users interact with this variable directly? If not would it not make sense to remove it entirely?

Reply:
Users would not interact with it, just so that the information is stored in the object and can be used internally. I want to ditch it as discussed, but would currently require too many code changes with the tests, easier when we first establish using mocks in the fixtures, will create a task for both things. Using `_info` right now basically establishes the behaviour of how it was before, not ideal, but better to do it in steps and you still get the `info` property. Should work for now (?).